crates/polars-arrow/src/array/binview/mod.rs

```diff
@@ -505,3 +517,31 @@ impl<T: ViewType + ?Sized> Array for BinaryViewArrayGeneric<T> {
         Box::new(self.clone())
     }
 }
+
+impl<T: ViewType + ?Sized> Splitable for BinaryViewArrayGeneric<T> {
+    fn check_bound(&self, offset: usize) -> bool {
+        offset <= self.len()
+    }
+
+    unsafe fn _split_at_unchecked(&self, offset: usize) -> (Self, Self) {
+        let (lhs_views, rhs_views) = unsafe { self.views.split_at_unchecked(offset) };
+        let (lhs_validity, rhs_validity) = unsafe { self.validity.split_at_unchecked(offset) };
+
+        (
+            Self::new_unchecked_unknown_md(
```

Comment:
I think we can copy the `slice` behavior here. We can clone the original `struct` and then set total byte len to `UNKNOWN`:

```rust
self.total_bytes_len.store(UNKNOWN_LEN, Ordering::Relaxed)
```